frontend/src/views/web/partials/sidebar.html

```diff
@@ -14,15 +14,15 @@
                 </li>
                 <li>
                     <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.teams" ui-sref-active="side-active-link">
-                        <i class="fa fa-users side-link-icon"></i> Teams</a>
+                        <i class="fa fa-users side-link-icon"></i> My Participant Teams</a>
```

Comment:
Or may be it's a My Participation Team section where only team in which a particular user is a member can only be listed here, is it?